app/config/services.yml

```diff
@@ -11,12 +11,12 @@ services:
             - '@repository.user'
 
     use_case.user_follow_user:
-        class: AppBundle\UseCase\MakeUserFollowUserUseCase
+        class: AppBundle\UseCase\FollowUser
         arguments:
             - '@repository.user'
 
     repository.user:
-        class: AppBundle\Repository\UserRepository
+        class: AppBundle\Repository\DoctrineUsers
```

Comment:
important thing is that this is doctrine implementation of UserRepository interface